src/crimson/osd/replicated_backend.cc

```diff
@@ -44,7 +44,7 @@ ReplicatedBackend::_submit_transaction(std::set<pg_shard_t>&& pg_shards,
 				       std::vector<pg_log_entry_t>&& log_entries)
 {
   LOG_PREFIX(ReplicatedBackend::_submit_transaction);
-  DEBUGDPP("object {}, {}", dpp, hoid);
+  DEBUGDPP("object {}", hoid);
```

Comment:
I think this does actually expect dpp, it's just the extra {} that needs to be removed.  See crimson/common/log.h, DEBUGDPP->LOGDPP.